web-app/helpers/certificate-designs.ts

```diff
@@ -1,17 +1,13 @@
 // https://dev.to/sudo_overflow/diy-generating-dynamic-images-on-the-fly-for-email-marketing-h51
-import {
-  Canvas,
-  // registerFont,
-  loadImage,
-} from 'canvas';
+import { Canvas, registerFont, loadImage } from 'canvas';
 
 // TODO: Update this section:
-export const width = 1160; // width of the image
-export const height = 653; // height of the image
-// const fontFile = './fonts/Sign-Painter-Regular.ttf';
-const fontFamily = 'signpainter';
+export const width = 1080; // width of the image
+export const height = 1080; // height of the image
+const fontFile = './fonts/Manrope-VariableFont_wght.ttf';
```

Comment:
If this is the only font file that we'll ever need, then maybe we should just delete the other ttf files from the repo.

Reply:
done